ddtrace/internal/buffer.py

```diff
@@ -1,75 +1,6 @@
-from collections import deque
-import threading
-from typing import Deque
-from typing import List
-
-import attr
-
-
 class BufferFull(Exception):
```

Comment:
Should we just move these since they're the only items in this module now? Probably into `encoding.py`?